### ISSUE TITLE
Dithering Sill Present on Weapons with r_softemu set to 3

### ISSUE THREAD

**Issue by OpenRift412** [andrei-drexler/ironwail#247]
I feel like if I don't make this issue it's gonna eat away at me lol

I noticed a weird discrepancy recently with software emulation that causes dithering to still appear present on weapons when r_softemu is set to 3. This is not accurate to how it appears in the vanilla software renderer.

Here are some pictures for comparison.

Ironwail:
![image](https://github.com/andrei-drexler/ironwail/assets/29780788/81766bcd-d921-4601-bc55-e72b59ad0451)

Vanilla SW Renderer:
![image](https://github.com/andrei-drexler/ironwail/assets/29780788/d0a0e09f-f9e5-478a-981a-7c1fc337d8d0)

If we could get a fix for this, it would be much appreciated.

**Comment by Terraformer9x** [comment]
I don't think it needs to be "fixed" as I imagine there are people who actually like the dithering. However it would be a better deal if dithering could be toggled via a cvar.

(A little side note that may or may not be relevant, I noticed that even with r_softemu set to 0, there still seems to be some sort of dithering shader albeit it's very faint but becomes more noticeable if you lower the render scale.)
